src/python/pants/backend/plugin_development/pants_requirements.py

```diff
@@ -26,21 +28,29 @@ class PantsRequirementsTestutilField(BoolField):
     help = "If true, include `pantsbuild.pants.testutil` to write tests for your plugin."
 
 
+class PantsRequirementsVersionField(StringField):
+    alias = "version"
+    default = PANTS_SEMVER.public
```

Comment:
Could you explain about more about when you're expecting someone to set this?

Reply:
Yup!

It's for people who want multiple Pants versions in different resolves (primarily for testing a plugin across several versions).

As an example: https://github.com/tgolsson/pants-backends/blob/main/pants-plugins/BUILD#L3-L19

Reply:
Ah, okay, so it's a new feature, that's independent of installing from GitHub releases vs. PyPI? (That's fine, just trying to understand the moving parts.)

Reply:
No it's very much tied into this initiative of getting off PyPI and using GitHub release wheels.

Right now people can specify easy requirements for older pants if they're testing against multiple versions.

Without PyPI that gets reeeeeeally hard, so in the same code that works for (this) version of pants we can help out with other versions

Reply:
Oh, right. I get it now (spelling it out, because I'm slow this morning): 
- `pants_requirements` was just a convenient form of those manual `python_requirement`s before, so one can easily choose different versions by writing out the `python_requirement`s manually (as done there)... 
- but, after this change, those requirements don't work quite as conveniently (one would need to add the appropriate `[python-repos] find_links = ...` values?)
- adding an explicit version here preserves the ease.

Nice. Makes sense.